.changeset/gentle-zoos-flash.md

```diff
@@ -0,0 +1,5 @@
+---
+'storybook': patch
```

Comment:
not 100% sure if this one needs a changeset (cc @backstage/reviewers)

Reply:
nop :)